app/messages.py

```diff
@@ -135,6 +135,8 @@
 
 # Login errors
 USER_ENTERED_INCORRECT_PASSWORD = {"message": "Current password is incorrect."}
+USER_ENTERED_CURRENT_PASSWORD = {"message": "New password should not be same "
+                                            "with the current password."}
```

Comment:
```suggestion
                                            "as the current password."}
```

Reply:
@isabelcosta Oh yes! Was kinda confused writing this! Will change that rn.